pkg/api/titond_api.go

```diff
@@ -27,3 +28,33 @@ func (t *TitondAPI) CreateNetwork(data *model.Network) *model.Network {
 
 	return result
 }
+
+func (t *TitondAPI) CreateL2Geth(data *model.Resource) {
+	// TODO : deal with DB
+	t.db.CreateResource()
+
+	namespace := "default"
+	createL2Geth(t.k8s, namespace)
+}
+
+func createL2Geth(client *kubernetes.Kubernetes, namespace string) {
+	obj := getObject("l2geth", "configMap")
```

Comment:
I think it is OK in a PoC project. In production, we may need more check and sync

Reply:
fix it! thank you

Reply:
That's great!